src/coreclr/jit/gentree.h

```diff
@@ -4037,6 +4048,16 @@ struct GenTreeField : public GenTreeUnOp
         return (gtFlags & GTF_FLD_VOLATILE) != 0;
     }
 
+    bool IsNeverNegative() const
+    {
+        return gtFldIsNeverNegative;
```

Comment:
`GenTreeField` also can be used for `GT_FLD_ADDR`, so same comment as for `GenTreeLclVar::IsNeverNegative` applies.

Reply:
Fixed.